### PR TITLE
Add external wrenches logging to yarp robot logger device

### PR DESCRIPTION
**Pull request by isorrentino**

### CODE REVIEW

File: devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h [hunk 1]
```diff
@@ -71,6 +73,9 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     bool m_streamJointStates{false};
     bool m_streamMotorPWM{false};
     bool m_streamPIDs{false};
+    bool m_streamWrenches{false};
+    bool m_streamFTs{false};
+    bool m_streamInertials{false};
```
Comment (GiulioRomualdi) [inline devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h:78]: Those parameters are not used 😄

File: devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h [hunk 2]
```diff
@@ -61,6 +62,7 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
 
     std::vector<std::string> m_IMUNames;
     std::vector<std::string> m_FTNames;
+    std::vector<std::string> m_CartesianWrenchNames;
```
Comment (GiulioRomualdi) [inline devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h:65]:
```suggestion
    std::vector<std::string> m_cartesianWrenchNames;
```

File: devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h (hunk 1 above)
Comment (isorrentino) [reply devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h:78]: Now yes :)

**Comment by isorrentino** [comment]
PR ready again for review.

File: devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h (hunk 1 above)
Comment (GiulioRomualdi) [reply devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h:78]: In theory, you do need them. Indeed if wrench are streamed the associated map is empty

Comment (isorrentino) [reply devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h:78]: Indeed I added the check to fill the map

https://github.com/isorrentino/bipedal-locomotion-framework/blob/55e8b428c2dddc4bf69f2a0df6e0cde028994349/devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp#L247-L332

**Comment by prashanthr05** [comment]
@GiulioRomualdi please proceed with the merge without my review, if you think the changes are good.

File: devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h (hunk 1 above)
Comment (isorrentino) [reply devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h]: Got your point. Changed again the code.

**Review by GiulioRomualdi — APPROVED** [review]

**Pull request merged**